timeless/templates/restaurants/reservations/create_edit.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
```

Comment:
@paulodamaso Take a look how do the same in this PR - https://github.com/timelesslounge/timelessis/pull/209/files
It's a flask way. If you can't do it now, let's leave a puzzles:
- add forms for view
- after forms implementation refactor template is that way - https://github.com/timelesslounge/timelessis/pull/209/files